docs/packages.rst

```diff
@@ -42,7 +42,7 @@ fv3config_ (`docs <https://fv3config.readthedocs.io/en/latest/>`_) provides rout
     readme_links/fv3fit_readme
     readme_links/loaders_readme
     readme_links/fv3viz_readme
-    readme_links/offline_ml_diags_readme
+    readme_links/offline_readme
```

Comment:
does this offline readme still exist somewhere?

Reply:
Oops, it got lost in the move... I put it back in the offline dir.